### PR TITLE
Feature/endpoint logging

### PR DESCRIPTION
**Pull request by bbangert**
Fixes Issue #67.

### CODE REVIEW

File: autopush/endpoint.py [hunk 1]
```diff
@@ -103,6 +120,7 @@ def _process_uaid(self, result):
         """Process the result of the AWS call"""
         if not result:
             self.set_status(404)
+            log.msg("UAID not found in AWS.", **self._client_info())
```
Comment (kitcambridge) [inline autopush/endpoint.py]: Should we log the `uaid` hash too?

Comment (bbangert) [inline autopush/endpoint.py]: Yup, patch incoming.

**Comment by kitcambridge** [comment]
Looks great!

**Comment by kitcambridge** [comment]
That's a very strange, unrelated failure. Looks like it tries to connect to AWS instead of using moto?

**Comment by bbangert** [comment]
Yea, quite odd, kicking the build.

**Comment by bbangert** [comment]
Looks fine now.

**Pull request merged**